code/datums/extensions/health/atom.dm

```diff
@@ -0,0 +1,411 @@
+/atom
+	/**
+	 * Whether or not the atom should use the health handler when initialized.
+	 */
+	var/use_health_handler = FALSE
+
+	/**
+	 * Initialization config for the health handler.
+	 *
+	 * `max_health` - Default maximum health.
+	 *
+	 * `resist_*` - Resistance multiplier for various damage types, where `*` is replaced by the damage type's string value.
+	 */
+	var/list/health_handler_config = list(
+		"max_health" = 100
+	)
+
+	/**
+	 * Ease-of-use holder for the health extension.
+	 */
+	var/datum/extension/health/health_handler
+
+	/**
+	 * Current health for simple health processing
+	 */
+	var/simple_health
+
+	/**
+	 * Maximum health for simple health processing
+	 */
+	var/simple_max_health
+
+
+/atom/Initialize(mapload, ...)
```

Comment:
Don't override Initialize on `/atom` itself, this is the level where that proc is defined.

Reply:
Moved.